Source/StandardLibrary.mjs

```diff
@@ -89,6 +89,13 @@ export let standardLibrary = (function() {
         }
         print();
 
+        for (let type of [`uchar`, `ushort`, `uint`, `char`, `short`, `int`, `half`, `float`]) {
```

Comment:
How about the standard library in the spec?

Reply:
Same change is in there :)